ansible/roles/util-pgpass/tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- name: place .pgpass
+  sudo: yes
+  template: src=postgres.pgpass.j2 dest=~/.pgpass mode=0600 backup=yes
```

Comment:
Good suggestion. We're using .pgpass within docker files, so I doubt if we'll ever want to put a .pgpass file in another user's home directory. I'll make the destination explicit.